src/server/services/csvPipeline/uploadMeters.js

```diff
@@ -64,4 +79,39 @@ async function uploadMeters(req, res, filepath, conn) {
 		});
 }
 
+/**
+ * Checks if the string is a valid GPS representation. This requires it to be two numbers
+ * separated by a comma and the GPS values to be within allowed values. The should be a latitude, longitude pair.
+ * This is very similar to src/client/app/utils/calibration.ts but not TypeScript and does not do popup.
+ * @param input The string to check for GPS values
+ * @returns true if string is GPS and false otherwise.
+ */
+function isValidGPSInput(input) {
+	if (input.indexOf(',') === -1) { // if there is no comma
+		return false;
+	} else if (input.indexOf(',') !== input.lastIndexOf(',')) { // if there are multiple commas
+		return false;
+	}
+	// Works if value is not a number since parseFloat returns a NaN so treated as invalid later.
+	const array = input.split(',').map(value => parseFloat(value));
```

Comment:
I think it's cleaner to say `.map(parseFloat)` here instead of `.map(value => parseFloat(value)`.

Reply:
I think this is a valid point. I checked the code base and found that there are a number of places that use the syntax in this PR and not the other way. To keep it consistent, I am going to leave it here but they all could be changed in the future.